TESTING-INSTRUCTIONS.md

```diff
@@ -2,6 +2,107 @@
 
 ## Unreleased
 
+### Payment gateway suggestions feature
+
+1. Note that the `payment-gateway-suggestion` feature should be turned on in your current build
+2. Navigate to the homescreen via WooCommerce -> Home
+3. Click on "Set up payments"
+
+#### How to test
+
+Individual payment gateway plugins dictate the settings and connection flow.  For testing purposes, we'll test both the default behavior of the gateway and the enhanced configuration behavior.
+
+1. On the payments task, click "Set up" or "Enable" next to a gateway
+2. Note that the gateway is installed if it requires a plugin
+3. On the connection step, a button should be presented that links to the gateway's (legacy) settings screen
+4. Delete the plugin
+5. Clone and checkout one of the gateway PRs in this issue - https://github.com/woocommerce/woocommerce-admin/issues/6878
+6. If setting fields are shown, make sure that validation works, input is saved, and is persisted on page refresh.  Make sure the gateway is marked as enabled and not labeled "Requires setup" if all fields are completed.
+7. If the "Connect" button is shown, follow the connection flow.  Make sure that you are returned to the payments task and that the gateway is enabled and marked as configured.
+8. Remove some settings manually under the payment gateway's legacy settings screen.  Make sure the gateway is no longer marked as configured.
+
+
+##### PayFast
+
+1. Set your country to South Africa in WooCommerce->Settings
+2. Don't select CBD as an industry during onboarding
+
+##### Paystack
+
+1. Set your country to South Africa in WooCommerce->Settings
+2. Don't select CBD as an industry during onboarding
+3. Complete the payment tasks
+4. Make sure that `ZAR` is the site currency after configuration of Paystack and that "test mode" is turned off in settings
+
+##### Stripe
+
+1. Set your store country to a Stripe supported country (e.g., US - https://stripe.com/global )
+2. Don't select CBD as an industry during onboarding
+3. Make sure you're using a site with https
+4. Attempt to use the oauth connection flow to enable the gateway, making sure you are returned to the site and connection is successful
+5. Remove the connection and use a non-https site
+6. Check that the manual settings configuration flow is shown
+
+##### PayPal
+
+1. Set your store country to any country except India
+2. Don't select CBD as an industry during onboarding
+3. Make sure the PayPal connection flow is shown and works as expected
+4. Throw an error in the REST endpoint `WC_PAYPAL_NAMESPACE + '/onboarding/get-params'`
```

Comment:
Instead of doing this from the code, could we use the `Network request blocking` feature in Chrome console for the `*/onboarding/get-params` url?